introduction/Pavit/Issue#37/main.cpp

```diff
@@ -0,0 +1,16 @@
+#include <iostream>
+
+using namespace std;
+
```

Comment:
Use proper indentation

Reply:
I have updated file with identation. Please review it again @AkshayCHD